libs/backend/src/scan/cast_vote_records/export.ts

```diff
@@ -349,7 +357,8 @@ export async function exportCastVoteRecordReportToUsbDrive(
     const exportReportResult = await exporter.exportDataToUsbDrive(
       reportDirectory,
       CAST_VOTE_RECORD_REPORT_FILENAME,
-      castVoteRecordReportStream
+      castVoteRecordReportStream,
+      { machineDirectoryToWriteToFirst }
```

Comment:
Definitely a bit clunky, but I think it's clear, and it also makes for the most minimal changes elsewhere in CVR export logic, which is very likely going to change after SLI anyway, to better accommodate continuous backup.

Other alternatives included:

- Explicitly calling both `exporter.exportData` and `exporter.exportDataToUsbDrive` everywhere that we need to write to both places. This would require lots of repeated code.
- Writing all files to disk and then copying the whole directory to USB. This would require either using the experimental `fs.cp` or writing our own code to recursively copy a directory.